R/dose_grid_search.R

```diff
@@ -247,23 +241,22 @@ dose_grid_search <- function(
 #' @param model model for simulating dose (estimation model)
 #' @param value element of the dose/interval grid
 #' @param grid_type either `dose` grid or `interval` grid
-#' @param t_obs time at which observation should be calculated
 #' @param obs Value of `obs` as determined by [dose_grid_search()] (i.e. either "obs" or AUC compartment)
 #' @md
-simulate_dose_interval <- function(value,
-                              grid_type = "dose",
-                              dose_update,
-                              regimen,
-                              md,
-                              pta,
-                              target_design,
-                              model,
-                              t_obs,
-                              omega,
-                              obs,
-                              ruv,
-                              ...) {
-
+simulate_dose_interval <- function(
```

Comment:
just formatting, and removing `t_obs` as argument (that is now calculated inside the function)